### ISSUE TITLE
[a11y] [Slash] MultiSelect reinit button is aria-hidden

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/design-system#1529]
On MultiSelect we have a button to reinitialize what we selected.

<img width="904" height="82" alt="Image" src="https://github.com/user-attachments/assets/c0701055-9c4a-485d-ac04-9ceddab9ba7f" />

But this button is actually a div and has the attribute aria-hidden which means it cannot be read and targeted by tests.

<img width="1042" height="110" alt="Image" src="https://github.com/user-attachments/assets/0e801947-26d9-4e37-ad71-7da7d26e23df" />
